### PR TITLE
Update Spyse module

### PR DESCRIPTION
**Pull request by DarkMristov**
Spyse v3 API has been deprecated.
See API references: https://spyse-dev.readme.io/reference/quick-start

This pull request will update the module for Spyse API v4 and add more events to the module.
New events that Spyse module will emit:

OPERATING_SYSTEM
COUNTRY_NAME
VULNERABILITY
COMPANY_NAME
WEBSERVER_TECHNOLOGY
IP_ADDRESS
IPV6_ADDRESS
DNS_SPF
DNS_TEXT
PROVIDER_DNS
DOMAIN_REGISTRAR
HTTP_CODE
EMAILADDR
SSL_CERTIFICATE_ISSUED
SSL_CERTIFICATE_ISSUER
WEB_ANALYTICS_ID

### CODE REVIEW

File: modules/sfp_spyse.py [hunk 1]
```diff
@@ -430,13 +440,144 @@ def handleEvent(self, event):
...
+                                evt = SpiderFootEvent('SSL_CERTIFICATE_ISSUER',
+                                                      domain_cert_summary_issuer["organization"], self.__name__,
+                                                      event)
+                                self.notifyListeners(evt)
+
+                    domain_trackers = domain_item.get("trackers")
+                    if domain_trackers:
+                        if domain_trackers.get("google_analytics_key"):
+                            evt = SpiderFootEvent('WEB_ANALYTICS_ID', domain_trackers["google_analytics_key"],
+                                                  self.__name__, event)
```
Comment (bcoles) [inline modules/sfp_spyse.py:578]: SpiderFoot uses a specific format for `WEB_ANALYTICS_ID`.

```suggestion
                            evt = SpiderFootEvent("WEB_ANALYTICS_ID", f"Google Analytics: {domain_trackers.get("google_analytics_key")}", self.__name__, event)
```

Comment (DarkMristov) [reply modules/sfp_spyse.py]: Thanks!

**Comment by smicallef** [comment]
This is great, thanks! Please take a look at the failed test though, looks like there's invalid syntax. Were you able to test this module beforehand?

**Comment by DarkMristov** [comment]
@smicallef Oh. Sorry. I accept changes from bcoles and forget to look at tests.

**Comment by DarkMristov** [comment]
@smicallef Can you give me a proper command to check code complexity, pls? Because I can't find it in the readme, or I miss something.

**Comment by smicallef** [comment]
Sure, they are all here: https://github.com/smicallef/spiderfoot/blob/master/.github/workflows/tests.yaml

**Pull request merged**